src/enochecker/checkerservice.py

```diff
@@ -303,12 +304,12 @@ def service_info() -> Dict[str, Union[str, int]]:
             return info_dict  # type: ignore # (mypy would infer Dict[str, object] instead)
 
         except Exception:
-            print("SERVICE INFO NOT SPECIFIED!!!11ELF!")
-            print(
+            logger.error("SERVICE INFO NOT SPECIFIED!!!11ELF!")
```

Comment:
How about a single log instead of multiple? That way it's easier to see in kibana I think.